Gemfile

```diff
@@ -4,7 +4,7 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '4.1.5'
+gem 'rails', '4.1.14.1'
```

Comment:
RPMs?  No, we'll continue to use 4.1.5 for those, similar to previous releases (3.2.22 or latest with gems or 3.2.8 with RPMs).  RPMs use bundler_ext which ignores this version spec, so it won't cause an error.